lib/interactivity-api.php

```diff
@@ -14,13 +14,24 @@ function gutenberg_reregister_interactivity_script_modules() {
 	wp_deregister_script_module( '@wordpress/interactivity' );
 	wp_deregister_script_module( '@wordpress/interactivity-router' );
 
+	$experiments                  = get_option( 'gutenberg-experiments' );
+	$full_page_navigation_enabled = isset( $experiments['gutenberg-full-page-client-side-navigation'] );
+
+	$index_path    = gutenberg_dir_path() . 'build/interactivity/index.min.asset.php';
+	$index_asset   = file_exists( $index_path ) ? require $index_path : null;
+	$index_version = isset( $index_asset['version'] ) ? $index_asset['version'] : $default_version;
+
```

Comment:
This needs conflict fixes from #65460. What are the changes to this file for? You may be able to remove all the changes here 🤞 

It looks like this is intended to remove a version from the `@wordpress/interactivity-router` package if full page navigation is enabled. I suspect that is because a time-based default was used before which would make the URL for the module not match across page loads. With #65460, a stable version hash should be used from the generated asset file. If the module changes, the hash will change, but that should only happen during active development and would require a reload of the module anyways.

Reply:
Thank you Jon! 

> It looks like this is intended to remove a version from the @wordpress/interactivity-router package if full page navigation is enabled. I suspect that is because a time-based default was used before which would make the URL for the module not match across page loads.

That's exactly right. I'll rebase and check if it works as expected now.

Reply:
I just tested it and indeed it works fine after #65460 was merged 👍